api/query/query.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package query
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	io "io"
+	"io/ioutil"
+	"log"
+	http "net/http"
+	"strconv"
+	"sync"
+	time "time"
```

Comment:
Here are some unnecessary import renamings as well. My local `gofmt` would get rid of them.